Makefile

```diff
@@ -214,6 +214,21 @@ image-build-iso: rpm
 iso: image-build-configure image-build-iso
 .PHONY: iso
 
+rpm-podman:
+	@if [ -z "$(shell podman images -q microshift-rpm-builder)" ]; then \
+		podman build \
+			-v /etc/pki/entitlement/:/etc/pki/entitlement \
+			-t microshift-rpm-builder - < ./hack/Containerfile.rpm-builder ; \
```

Comment:
Do we want to include a version in the image name, i.e. `microshift-4.13-rpm-builder-rhel8-go1.19`? I mean, we should be able to easily cross-compile from different branches on the same machine now, but images may be different.